File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.js
const input = require("fs").readFileSync(process.platform === "linux" ? "/dev/stdin" : "알고리즘/example.txt").toString().trim().split("\n").map((el) => el.split(" ").map(Number));
const N = input[0][0];
const X = input[0][1];
const test_case = [...input[1]];
let answer = '';

function smaller_X(arr){
    for(let i = 0; i < N; i++){
        if(arr[i] < X){
            answer += arr[i] + " ";
        }
    }
    console.log(answer);
}

smaller_X(test_case)

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.py
N, X = [int(x) for x in input().split(" ")]

list = []
data = input().split(" ")

for item in data:
    list.append(int(item))    

answer = ''

for item in list:
    if item < X :
        answer += str(item) + " "

print(answer)